modules/ossm-observability-access.adoc

```diff
@@ -6,13 +6,15 @@
 [id="ossm-observability-access-console_{context}"]
 = Accessing the Kiali console
 
-To access the console, in the menu bar, click the *Application launcher* > *Kiali*.
+To access the console, you must have the Kiali operator installed and a deployed an application in a project.
 
-1. In the {product-title} menu bar, click the *Application launcher* > *Kiali*.
+1. Use the perspective switcher to switch to the Developer perspective. 
 
-2. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console.
+2. Click *Topology*.
```

Comment:
For topology, eligible topology nodes will offer a link.  For Admin, eligible projects will have link.